src/index.ts

```diff
@@ -108,35 +108,33 @@ function noneSort(_a: string, _b: string): number {
 /**
  * Sorting algorithms for categories in a custom sort order definition.
  */
-enum CategorySort {
-  CaseInsensitiveLexical = 'caseInsensitiveLexical',
-  CaseInsensitiveNumeric = 'caseInsensitiveNumeric',
-  CaseInsensitiveReverseLexical = 'caseInsensitiveReverseLexical',
-  CaseInsensitiveReverseNumeric = 'caseInsensitiveReverseNumeric',
-  Lexical = 'lexical',
-  Numeric = 'numeric',
-  ReverseLexical = 'reverseLexical',
-  ReverseNumeric = 'reverseNumeric',
-  None = 'none',
-}
+type CategorySort =
```

Comment:
Did you mean to export this as well?

```suggestion
export type CategorySort =
```

Reply:
I didn't export it because you now get autocomplete with the string union. That said, there's no harm in exporting it, and someone may find it useful.